src/components/SimplePopup/index.js

```diff
@@ -76,6 +78,7 @@ class SimplePopup extends React.Component<PropsShape> {
   listenerAction = (event: Object) => {
     if (!this.reactOverlay || !this.reactPopup) return null;
     if (
+      !this.props.cannotOutsideClickClose &&
```

Comment:
would you say it makes more sense if we prevent the listener from starting with this prop?

Reply:
yea that does makes sense. made the change